Makefile

```diff
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.12.0
```

Comment:
@mjlshen You'll also need to update the image tag in `build/`

Reply:
Thank you @morgante , I didn't know that. I've done that as well!